.github/workflows/build-electron.yml

```diff
@@ -25,23 +29,33 @@ jobs:
       run: npm install
 
     - name: Fetch InVEST Binaries
-      shell: bash
       run: npm run fetch-invest
 
-    - name: Build & Release Electron application
-      uses: samuelmeuli/action-electron-builder@v1
-      if: ${{ always() }}
-      with:
-        build_script_name: build
-        args: -c.extraMetadata.main=build/main.js
-        github_token: ${{ secrets.github_token }}
-        # If the commit is tagged with a version (e.g. "v1.0.0"),
-        # release the app after building
-        release: ${{ startsWith(github.ref, 'refs/tags/v') }}
+    - name: NPM Install
+      run: npm install
+
+    - name: Run the build script
+      run: npm run build
+     
+    - name: Run electron-builder on macOS
+      if: matrix.os == 'macos-latest'
+      env:
+        GH_TOKEN: env.GITHUB_TOKEN
+        DEBUG: electron-builder
+      run: |
+        # use the built version as the entrypoint
+        # make sure we don't publish to github releases
+        npx --no-install electron-builder build --mac -c.extraMetadata.main=build/main.js --publish never
+
+    - name: Run electron-builder on Windows
+      if: matrix.os == 'windows-latest'
+      env:
+        GH_TOKEN: env.GITHUB_TOKEN
+        DEBUG: electron-builder
+      run: npx --no-install electron-builder build --windows -c.extraMetadata.main=build/main.js --publish never
```

Comment:
I just remembered that we also have `npm run dist` and `npm run pack` in our package.json. I use `npm run dist` regularly on Windows (I never use `pack` and forget what the extra `--dir` flag does). Do you think it's useful to call `npm run dist` here? Or are the options we need in GHA different from what we would use locally? 

I think leaving out the `--windows` or `--mac` flags will default to building for the current platform. So it might be possible to consolidate these two OS-specific steps into to a single "Run electron-builder" step. But if you prefer being explicit with the OS flag, that's fine by me also!

Reply:
ah, good idea! The `--dir` option is: "Build unpacked dir. Useful to test." I took `pack` out of `package.json` since we aren't using it, and added `--publish never` to the `dist` script. Also explicitly included the `build` command. I think leaving out the platform flag is fine too.